added a fallback example and added descriptions of the previous examples (#29460)
mdn/content: files/en-us/web/css/attr/index.md

### content property

> This example we are prepending the value of the `data-foo` [`data-*`](/en-US/docs/Web/HTML/Global_attributes/data-*) [global attribute](/en-US/docs/Web/HTML/Global_attributes) from the {{HTMLElement("p")}} element.

Reviewer: ```suggestion
In this example, we prepend the value of the `data-foo` [`data-*`](/en-US/docs/Web/HTML/Global_attributes/data-*) [global attribute](/en-US/docs/Web/HTML/Global_attributes) to the contents of the {{HTMLElement("p")}} element.
```

{{SeeCompatTable}}

> This example we are setting the value of {{CSSXRef("background-color")}} with the value of `data-background` [`data-*`](/en-US/docs/Web/HTML/Global_attributes/data-*) [global attribute](/en-US/docs/Web/HTML/Global_attributes) from the {{HTMLElement("div")}} element.

Reviewer: ```suggestion
In this example, we set the CSS value of {{CSSXRef("background-color")}} to the value of the `data-background` [`data-*`](/en-US/docs/Web/HTML/Global_attributes/data-*) [global attribute](/en-US/docs/Web/HTML/Global_attributes) assigned to the {{HTMLElement("div")}} element.
```

{{EmbedLiveSample("color_value", "100%", 50)}}

### using fallback

{{SeeCompatTable}}

> This example we are appending the value of `data-browser` [`data-*`](/en-US/docs/Web/HTML/Global_attributes/data-*) [global attribute](/en-US/docs/Web/HTML/Global_attributes) from the {{HTMLElement("p")}} element, if it is missing we _fallback_ to **Unknown**.

Reviewer: ```suggestion
In this example, we append the value of `data-browser` [`data-*`](/en-US/docs/Web/HTML/Global_attributes/data-*) [global attribute](/en-US/docs/Web/HTML/Global_attributes) to the {{HTMLElement("p")}} element. If the `data-browser` attribute is missing from the {{HTMLElement("p")}} element, we append the _fallback_ value of "**Unknown**".
```
